src/elements/e-table.vue

```diff
@@ -93,21 +245,80 @@
        *
        * @property {String} title - Column title.
        * @property {String} [key] - Column identifier on the rows.
-       * @property {String} [titleHidden = false] - Title visibility state.
+       * @property {Boolean} [titleHidden = false] - Title visibility state.
        * @property {String} [slotName] - Dynamic slot name for optional cell templates.
        * @property {String} [align = 'left'] - A flag to set the column alignment (default: 'left', options: 'center', 'right').
        * @property {Boolean} [sortable = false] - A flag that specifies whether the column is sortable or not.
        * @property {Function} [sort = (a, b) => a > b] - A custom sort function that might be passed for each column.
        * @property {Function} [onClick] - Allows to define a click handler for each cell.
-       *
        */
       columns: {
         type: Array,
         required: true,
       },
+
+      /**
+       * Accepts a method to generate a link for each row (except for columns with 'onClick' callback).
+       *
+       * @param {Object} rowLink - A definition for the row link.
```

Comment:
Probably a general thing we can discuss: but I would prefer to have no JSDoc for these things anymore. I created interfaces in ADM, that way the information is also always reusable.

Reply:
Will do in Vue 3